smtp/connection.ts

```diff
@@ -554,7 +555,7 @@ export class SMTPConnection extends EventEmitter {
 	 * @returns {boolean} whether the extension exists
 	 */
 	public has_extn(opt: string) {
-		return (this.features ?? {})[opt.toLowerCase()] === undefined;
+		return this.features != null && this.features[opt.toLowerCase()] != null;
```

Comment:
@eleith this behavior has been in place since before i even showed up (see 50b9ec18e83e3f9f4dec432bef314205248f561a) but it just seems _wrong_

Reply:
https://github.com/eleith/emailjs/commit/288ec6bae9c83f8b7353b271210c79f66c881a32#diff-e24a4bfef4475b3b5d01828e660b3042f4fa02502c09864cd104c4418b235e4f

Reply:
it took a bit of digging because this file has been renamed more than once.

Reply:
the only change i did to that method in that commit was to add nullish coalescing. i'm referring specifically to the comparison to `undefined`.

Reply:
agreed. it's wrong. i also can't find any use of this public method anywhere.